src/components/App.css

```diff
@@ -0,0 +1,44 @@
+#app {
+  display: flex;
+  width: 700px;
+  flex-direction: column;
+}
+
```

Comment:
You can center the calculator too, so that it appears in middle
` margin: auto;`